pkg/model/awsmodel/api_loadbalancer.go

```diff
@@ -266,6 +272,40 @@ func (b *APILoadBalancerBuilder) Build(c *fi.ModelBuilderContext) error {
 		}
 	}
 
+	// Allow Kops-controller from nodes to ELB, thence to the master instances.
+	if b.UseKopsControllerForNodeBootstrap() && b.Cluster.Spec.API != nil && b.Cluster.Spec.API.LoadBalancer != nil && b.Cluster.Spec.API.LoadBalancer.UseForInternalApi {
+		nodeGroups, err := b.GetSecurityGroups(kops.InstanceGroupRoleNode)
+		if err != nil {
+			return err
+		}
+
+		for _, nodeGroup := range nodeGroups {
+			suffix := nodeGroup.Suffix
+			c.AddTask(&awstasks.SecurityGroupRule{
+				Name:          fi.String(fmt.Sprintf("kops-controller-node%s-to-elb", suffix)),
+				Lifecycle:     b.SecurityLifecycle,
+				FromPort:      fi.Int64(wellknownports.KopsControllerPort),
```

Comment:
Any reason why the source port is set? I would think this one would be random.

Reply:
I think "from" and "to" here just means "beginning" and "end" of the port range, inclusive on both ends. It's confusing phrasing, no doubt.

Reply:
Damn... I always miss this part. Thanks for the reminder @seh .